org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/ScalaSourceFileEditor.scala

```diff
@@ -439,7 +440,35 @@ class ScalaSourceFileEditor
       else
         super.requestWidgetToken(requester, priority)
   }
+  private var outlinePage:ScalaOutlinePage = null
+  def getOutlinePage = outlinePage
+  override def getAdapter( required:Class[_]):AnyRef={
```

Comment:
When is the outline view not available? In case it is closed? Is `getAdapter` never called in this case?